packages/fiori/src/TimelineItem.ts

```diff
@@ -46,32 +44,28 @@ const LARGE_LINE_WIDTH = "LargeLineWidth";
  * <b>Note:</b> The event will not be fired if the <code>name-clickable</code>
  * attribute is not set.
  *
- * @event sap.ui.webc.fiori.TimelineItem#name-click
  * @public
  */
 @event("name-click")
-class TimelineItem extends UI5Element implements ITabbable {
+class TimelineItem extends UI5Element implements ITabbable, ITimelineItem {
 	/**
 	 * Defines the icon to be displayed as graphical element within the <code>ui5-timeline-item</code>.
 	 * SAP-icons font provides numerous options.
 	 * <br><br>
 	 *
 	 * See all the available icons in the <ui5-link target="_blank" href="https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html">Icon Explorer</ui5-link>.
 	 *
-	 * @type {string}
-	 * @name sap.ui.webc.fiori.TimelineItem.prototype.icon
-	 * @defaultvalue ""
+	 * @default ""
 	 * @public
+	 * @slot {Node[]} default - Determines the description of the <code>ui5-timeline-item</code>.
```

Comment:
this slot should be defined in class comment, not on property

Reply:
Done